src/Dockerfile.fedora

```diff
@@ -46,6 +46,10 @@ COPY root/usr/libexec/fix-permissions /usr/libexec/fix-permissions
 # to make sure of that.
 RUN INSTALL_PKGS="rsync tar gettext bind-utils postgresql-server postgresql-contrib nss_wrapper " && \
     INSTALL_PKGS+="findutils python2 xz" && \
```

Comment:
Shouldn't `python2` be replaced with `python3`, or is it still necessary?

Reply:
Thanks for poiting this out. I am actually not sure if we could not get away with python3 now, but considering rhel8 vrersion does not use python2 anymore as far as I can see there should not be a problem.
Will try out.